projects/storefrontapp-e2e/src/custom-locators.ts

```diff
@@ -1,4 +1,4 @@
-import { ProtractorBy, Locator } from 'protractor'; // tslint:disable-line
+import { ProtractorBy } from 'protractor'; // tslint:disable-line
```

Comment:
I'm not sure, but we probably don't need that tslint disable line if we remove Locator from import

```suggestion
import { ProtractorBy } from 'protractor';
```